packages/flutter/lib/src/widgets/color_filter.dart

```diff
@@ -12,6 +12,43 @@ import 'framework.dart';
 /// Applies a [ColorFilter] to its child.
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=F7Cll22Dno8}
+///
+/// {@tool dartpad --template=stateless_widget_scaffold}
+///
+/// The sample shows how to filter images with selected color
+/// and blending them with different modes
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return  Column(
+///     children:[
+///       ColorFiltered(
+///         colorFilter: ColorFilter.mode(
+///           Colors.red,
+///           BlendMode.modulate,
+///         ),
+///         child:Image.network('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg'),
+///       ),
+///       ColorFiltered(
+///         colorFilter: ColorFilter.mode(
+///           Colors.grey,
+///           BlendMode.saturation,
+///         ),
+///         child:Image.network('https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg'),
+///       ),
+///     ],
+///   );
```

Comment:
Since can overflow, I would consider wrapping this in a SingleChildScrollView:

```suggestion
///     return SingleChildScrollView(
///       child: Column(
///         children: [
///           ColorFiltered(
///             colorFilter: ColorFilter.mode(
///               Colors.red,
///               BlendMode.modulate,
///             ),
///             child: Image.network(
///                 'https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg'),
///           ),
///           ColorFiltered(
///             colorFilter: ColorFilter.mode(
///               Colors.grey,
///               BlendMode.saturation,
///             ),
///             child: Image.network(
///                 'https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg'),
///           ),
///         ],
///       ),
///     );
```